fastos/src/vespa/fastos/linux_file.cpp

```diff
@@ -44,7 +44,18 @@ FastOS_Linux_File::readInternal(int fh, void *buffer, size_t length, int64_t rea
 ssize_t
 FastOS_Linux_File::readInternal(int fh, void *buffer, size_t length)
 {
-    return File_RW_Ops::read(fh, buffer, length);
+    char * data = static_cast<char *>(buffer);
+    ssize_t read(0);
```

Comment:
Consider a different name for the variable, e.g. `has_read`

Reply:
Fixed